server/server.go

```diff
@@ -59,6 +60,12 @@ func New(db db.SQLClient, authority auth.Authenticator) *echo.Echo {
 	e.DELETE("/pricing_plans/:pricing_plan_id", auth.AdminOnly(api.DestroyPricingPlan(db)))
 	e.POST("/seed_pricing_plans", auth.AdminOnly(api.CreateMissingPricingPlans(db)))
 
+	// Repair events
+	if cf != nil {
+		e.GET("/repair_events", auth.AdminOnly(api.GetRepairedEvents(db, cf)))
+		e.POST("/repair_events", auth.AdminOnly(api.RepairEvents(db, cf)))
```

Comment:
when I call this with curl (the POST endpoint) I get into an infinite redirect loop to /repair_events

Reply:
I blame `curl -X POST` behaviour. Try using `-d ""` instead.